### PR TITLE
Nick/integrate backend

### PR DESCRIPTION
**Pull request by niwong**
## What's new in this PR?
[//]: # "#####################################"
[//]: # "####### YOUR ANSWER BELOW ###########"

Implement a `fetch` POST request to the `/invite` route on the backend sending the `RECORD_ID` of the pledge invitation just created.

[//]: # "############## END ##################"

### How to Review
[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # "#####################################"
[//]: # "####### YOUR ANSWER BELOW ###########"

[//]: # "############## END ##################"

### Related PRs
[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # "#####################################"
[//]: # "####### YOUR ANSWER BELOW ###########"

[//]: # "############## END ##################"

### Next steps
[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # "#####################################"
[//]: # "####### YOUR ANSWER BELOW ###########"

[//]: # "############## END ##################"

### Tests Performed, Edge Cases
[//]: # "#####################################"
[//]: # "####### YOUR ANSWER BELOW ###########"

[//]: # "############## END ##################"

### Screenshots
[//]: # "#####################################"
[//]: # "#### YOUR SCREENSHOTS BELOW ########"

[//]: # "############## END ##################"
CC: @dfangshuo

### CODE REVIEW

File: src/screens/admin/AdminDashboard.js [hunk 1]
```diff
@@ -75,17 +75,37 @@ class AdminDashboard extends React.Component {
...
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        pledgeInviteID
+      })
+    })
+      .then(res => res.json())
```
Comment (aivantg) [inline src/screens/admin/AdminDashboard.js]: Use async/await!

File: src/screens/admin/AdminDashboard.js [hunk 2]
```diff
@@ -75,17 +75,37 @@ class AdminDashboard extends React.Component {
...
         status: 'An error occurent when sending the invitation.'
       });
-    } else {
-      this.setState({
-        status: 'Successfully sent invitation.'
-      });
     }
+
+    // POST to backend to trigger email.
+    fetch('http://localhost:3001/invite', {
```
Comment (aivantg) [inline src/screens/admin/AdminDashboard.js]: use the "BACKEND_URL" constant in `constants.js` (I created one on my new branch) and append `.invite/`

Comment (aivantg) [reply src/screens/admin/AdminDashboard.js]: Feel free to create that constant, the URL here should be something like `Constants.BACKEND_URL` + '/invite'

**Comment by aivantg** [comment]
I think you can pass the tests if you pull from master? seems like an old bug

File: src/screens/admin/AdminDashboard.js (hunk 2 above)
Comment (aivantg) [inline src/screens/admin/AdminDashboard.js]: we can add this to `adminUtils` file now that i think about it
